### ISSUE TITLE
Pass `seed` and `batch_size` to the dataloader functions in `TabularDataModule`

### ISSUE THREAD

**Issue by BirkhoffG** [BirkhoffG/cfnet#39]
1. Pass `seed` and `batch_size` to `TabularDataModule.train_dataloader`, `TabularDataModule.val_dataloader`, and `TabularDataModule.test_dataloader`.

2. `batch_size` should also be an argument in `TrainingConfigs`
https://github.com/BirkhoffG/cfnet/blob/2ee1a3203a9935e89b2ed8adf175ee1150fd9960/cfnet/train.py#L15

3. Deprecated `batch_size` in `DataConfigs`

4. Finally, pass appropriate arguments:
https://github.com/BirkhoffG/cfnet/blob/2ee1a3203a9935e89b2ed8adf175ee1150fd9960/cfnet/train.py#L58-L59
